tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -108,10 +113,7 @@ class BrainTumorSegmentationPrediction(SegmentationPrediction):
 
 class CoCoInstanceSegmentationRepresentation(SegmentationRepresentation):
     def __init__(self, identifier, mask, labels):
-        try:
-            # pylint: disable=W0611
-            import pycocotools.mask as maskUtils
-        except ImportError:
+        if not maskUtils
             raise ValueError('can not create representation')
```

Comment:
It's not safe for static method usage. Please add check that maskUtils not None in `_convert_mask()`

Reply:
Please explain.